cis194/week8/tien/app/Main.hs

```diff
@@ -1,4 +1,14 @@
 module Main where
 
+import Party
+import System.Directory
+import System.IO
+
 main :: IO ()
-main = putStrLn "no use"
+main = do
+  dir <- getCurrentDirectory
+  handle <- openFile (dir ++ "/src/company.txt") ReadMode
+  contents <- hGetContents handle
+  putStrLn $ "Total Fun: " ++ (show . totalFun . read $ contents)
+  mapM_ putStrLn . totalEmpNameList . read $ contents
+  hClose handle
```

Comment:
I think we can still move some stuff out and only keep `IO` related things here.